### ISSUE TITLE
about how to calculate the average result

### ISSUE THREAD

**Issue by liuchangshiye** [cchen-cc/SIFA#17]
I am just a student, so the problem may be a little stupid. The result I got from the program is better than that of the paper. I think the reason is I don't know how to calculate the average value of the result. Should I take the result of the model with 20,000 iterations or take the average result of all the 20,000 models?

**Comment by cchen-cc** [comment]
We took the best model within the 20k iterations. 
It is possible to obtain better results than that reported in the paper as there are some randomness for the training.

**Comment by liuchangshiye** [comment]
OK. Thanks.

**Comment by chke097** [comment]
> I am just a student, so the problem may be a little stupid. The result I got from the program is better than that of the paper. I think the reason is I don't know how to calculate the average value of the result. Should I take the result of the model with 20,000 iterations or take the average result of all the 20,000 models?

您好，请问您是怎么成功跑出代码的？我一直遇到这个问题,不知道该怎么解决：aused by op 'shuffle_batch', defined at:
  File "main.py", line 626, in <module>
    main(config_filename='./config_param.json')
  File "main.py", line 622, in main
    sifa_model.train()
  File "main.py", line 355, in train
    self.inputs = data_loader.load_data(self._source_train_pth, self._target_train_pth, True)
  File "/data2/xxb/SIFA-master/SIFA-master/data_loader.py", line 112, in load_data
    1,50,10,num_threads=4)
  File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/training/input.py", line 1300, in shuffle_batch
    name=name)
  File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/training/input.py", line 846, in _shuffle_batch
    dequeued = queue.dequeue_many(batch_size, name=name)
  File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/ops/data_flow_ops.py", line 476, in dequeue_many
    self._queue_ref, n=n, component_types=self._dtypes, name=name)
  File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 3480, in queue_dequeue_many_v2
    component_types=component_types, timeout_ms=timeout_ms, name=name)
  File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
    return func(*args, **kwargs)
  File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
    op_def=op_def)
  File "/data2/xxb/anaconda/envs/tensorflow10/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1717, in __init__
    self._traceback = tf_stack.extract_stack()

OutOfRangeError (see above for traceback): RandomShuffleQueue '_2_shuffle_batch/random_shuffle_queue' is closed and has insufficient elements (requested 1, current size 0)
         [[Node: shuffle_batch = QueueDequeueManyV2[component_types=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](shuffle_batch/random_shuffle_queue, shuffle_batch/n)]]
